pkg/minikube/download/download.go

```diff
@@ -60,7 +61,7 @@ func CreateDstDownloadMock(src, dst string) error {
 // download is a well-configured atomic download function
 func download(src string, dst string) error {
 	var clientOptions []getter.ClientOption
-	if out.IsTerminal(os.Stdout) {
+	if out.IsTerminal(os.Stdout) && !(detect.GithubActionRunner()) {
```

Comment:
@spowelljr added the condition for using progress bar when terminal is not in Github Actions.